dao/src/main/resources/db/migration/V93__projectsPackagesCascadeDelete.sql

```diff
@@ -0,0 +1,23 @@
+-- This migration script modifies the foreign key constraints of the tables that reference projects and packages
+-- to add an ON DELETE CASCADE clause.
+
+ALTER TABLE packages_declared_licenses
```

Comment:
Just for my understanding: As these tables define m:n relations between packages/projects and licenses, the effect of this cascade constraint is that only rows in these tables are removed automatically, but no license entities. Is this correct? Maybe add some more explanation to the comment, so that this is directly obvious.